package.json

```diff
@@ -11,9 +11,8 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
-    "bootstrap": "^4.0.0-beta",
-    "edx-bootstrap": "^0.1.7",
-    "paragon": "edx/paragon",
+    "classnames": "^2.2.5",
+    "paragon": "edx/paragon#thallada/edx-bootstrap",
```

Comment:
I'll change this back to master once the PR is merged. Actually, I probably should make this an NPM link...